Meme-ify-Me/app/src/main/res/layout/activity_main.xml

```diff
@@ -6,16 +6,92 @@
     android:paddingTop="@dimen/activity_vertical_margin"
     android:paddingBottom="@dimen/activity_vertical_margin" tools:context=".MainActivity">
 
-    <TextView android:text="@string/hello_world" android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:id="@+id/textView" />
+
 
     <ImageView
+        android:layout_width="fill_parent"
+        android:layout_height="250dp"
+        android:src="@drawable/successkid"
+        android:layout_gravity="center"
+        android:id="@+id/image"/>
+
+
+    <Button
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:id="@+id/imageView"
-        android:layout_below="@+id/textView"
-        android:layout_centerHorizontal="true"
-        android:layout_marginTop="93dp" />
+        android:text="Take a new picture"
+        android:id="@+id/camera_button"
+        android:layout_below="@id/image"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentStart="true" />
+
+    <Button
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Choose a picture from gallery"
+        android:id="@+id/pic_from_gallery"
+        android:layout_below="@+id/camera_button"
+        android:layout_alignParentLeft="true"
+        android:layout_alignParentStart="true" />
+
+    <EditText
```

Comment:
This looks great, guys! I think the "choose meme type" edittext might be better as a TextView just because edittext would pop the keyboard up and could inadvertently let the user alter the text. we can also store "Choose Meme Type" as a string value, but not necessary.